addons/purchase/migrations/11.0.1.2/post-migration.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# © 2017 bloopark systems (<http://bloopark.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(
+        env.cr, 'purchase', 'migrations/11.0.1.0/noupdate_changes.xml',
```

Comment:
Fix version number. This is why travis is red.

Reply:
Copy Paste Issue, thanks. 👍